src/app/home-page/home-page.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component } from '@angular/core';
+import { IFeature } from '../module/feature';
+
+@Component({
+  selector: 'app-home-page',
+  templateUrl: './home-page.component.html',
+  styleUrls: ['./home-page.component.scss'],
+})
+export class HomePageComponent {
+  pageTitle = 'Home Page';
+  titleColor= "red";
+  cardTitle = 'Card for Organizing Activities';
+  features: IFeature[] = [
```

Comment:
this data should be entered here:  [features]="features" 
in home-page.component.html